Specs/DateTime/English/DateExtractor.json

```diff
@@ -21,6 +21,28 @@
       }
     ]
   },
+  {
+    "Input": "i'll go back april22",
```

Comment:
Build is broken. Don't forget to add the NotSupported clause for the other platforms (or to implement the change there too, if small).

Reply:
Added tags